app/encryption.py

```diff
@@ -50,6 +64,10 @@ def verify(self, to_verify: str | bytes) -> Any:
         except BadSignature:
             return self.serializer.loads(to_verify, salt=self.dangerous_salt)
 
+    # This ignores the signature so should be used with caution
```

Comment:
Let's put this comment as a docstring too please. 🙏 Along with parameters documentation.